app/frontend/components/pages/login/hardwareAuth.tsx

```diff
@@ -1,14 +1,29 @@
-import {h} from 'preact'
+import {Fragment, h} from 'preact'
 import {CRYPTO_PROVIDER_TYPES} from '../../../wallet/constants'
 import {TrezorLogoWhite, LedgerLogoWhite} from '../../common/svg'
 import {ADALITE_CONFIG} from '../../../config'
 import tooltip from '../../common/tooltip'
+import {connect} from '../../../helpers/connect'
+import actions from '../../../actions'
+import {useState, useCallback} from 'preact/hooks'
+import {localStorageVars} from '../../../localStorage'
 
 interface Props {
   loadWallet: any
 }
 
 const LoadByHardwareWalletSection = ({loadWallet}: Props) => {
+  const [dontBulkExportPubKeys, setBulkExport] = useState(
```

Comment:
I'd prefer to name it something like `disableBulkPublicKeyExport`

Reply:
I used negation in localstorage variable name sucne by default `window.localStorage.getItem(localStorageVars.BULK_EXPORT) === 'true'` is `false`. Anyway, you are right, a negated the expression and now it makes more sense. Thanks. 

```
  const [enableBulkExport, setBulkExport] = useState(
    window.localStorage.getItem(localStorageVars.BULK_EXPORT) !== 'true'
  )
  const toggleBulkExport = useCallback(
    () => {
      window.localStorage.setItem(localStorageVars.BULK_EXPORT, `${enableBulkExport}`)
      setBulkExport(!enableBulkExport)
    },
    [enableBulkExport]
  )